tablet_qt/tasks/cisr.cpp

```diff
@@ -3625,25 +3628,29 @@ QVector<QString> Cisr::panicSymptomFieldnames() const
     return fieldnames;
 }
 
```

Comment:
I'd delete the commented-out code or disable it with a preprocessor directive if it's likely to be needed in the future.